Delegates_And_Events_Task/MentorTask1/MentorTask1/obj/Debug/MentorTask1.csproj.CoreCompileInputs.cache

```diff
@@ -0,0 +1 @@
+7f4b213b428f4c013f19137338418ee1f5525793
```

Comment:
First learn about Git Ignore, and all the unwanted files and folders such as bin and obj and any user specific files should be added to the .gitignore file. Only after this is done, I will do the review.

Reply:
I have also clearly told you to follow Coding standards, I don't see the being followed. For example, naming conventions.